src/ore/src/tracing.rs

```diff
@@ -358,7 +352,7 @@ where
         });
         (Some(layer), reloader)
     } else {
-        let reloader = Arc::new(move |_| bail!("OpenTelemetry is disabled"));
+        let reloader = Arc::new(|_| Ok(()));
```

Comment:
as a followup, can we move the contents of this module into `mz-tracing`?

Reply:
oh yeah good call -- follow-up PR?

Reply:
yes, perfect!